Tools/interface/TrackRescaler.h

```diff
@@ -34,11 +34,14 @@ namespace jmt {
       double dxydszcov_;
     };
 
-    TrackRescaler() : enable_(false), era_(0), which_(w_JetHT) {}
-    void setup(bool enable, int era, int which) { enable_ = enable; era_ = era; which_ = which; }
+    //track rescaler takes in not only a track's pt and eta, but also the type 
+    // ie if it is a general track vs. muon or electron candidate 
+    TrackRescaler() : enable_(false), era_(0), which_(w_BTagDispJet), type_("") {}
```

Comment:
I am a little worrid that defaulting to the btag one could run us into issues if we aren't careful. Could we default it to w_max instead? I think then we will get an error if we fail to configure it (or perhaps we should even throw an exception in that case).